client/components/select-vertical/suggestion-search.tsx

```diff
@@ -33,18 +35,26 @@ const SelectVerticalSuggestionSearch: FC< Props > = ( {
 	const toggleIconRef = useRef( null );
 	const translate = useTranslate();
 
+	const showSuggestions = () => {
```

Comment:
technically these functions aren't necessary anymore, but I think they are still more readable 👍

Reply:
Yeah, I think it's useful. I also wrote a hook named `useBoolean` before, and it returns an array as followed 😆
```js
const [
  // the current value
  value,
  // a function to set the value to true
  enable,
  // a function to set the value to false
  disable,
] = useBoolean( initialValue );
```